app/src/main/java/com/framgia/forder/data/source/local/MyFirebaseMessagingService.java

```diff
@@ -21,32 +22,43 @@
 
 public class MyFirebaseMessagingService extends FirebaseMessagingService {
 
-    private static final String TAG = "MyFirebaseMsgService";
+    private static final String PREF_NOTIFICATION_NUMBER = "PREF_NOTIFICATION_NUMBER";
+    private static final String MESSAGE = "message";
+    private static int NUMBER_NOTIFICATION = 0;
```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName